libraries/api-extractor/src/ApiDefinitionReference.ts

```diff
@@ -14,7 +16,7 @@
  * \@microsoft/sp-core-library:Guid.equals
  * es6-collections:Map
  */
-export interface IApiDefinintionReferenceParts {
+export interface IApiDefinitionReferenceParts {
```

Comment:
Yeah "Definition" was misspelled.

---
In reply to: [129390968](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129390968) [](ancestors = 129390968)